lib/accumulate_distribute/util/generate_order.js

```diff
@@ -4,6 +4,7 @@ const _isFinite = require('lodash/isFinite')
 const _isObject = require('lodash/isObject')
 const { Order } = require('bfx-api-node-models')
 const genCID = require('../../util/gen_client_id')
+const { nBN } = require('@bitfinex/lib-js-util-math')
```

Comment:
this is part the core functionality of the algo. 

would be good to have two tests: one that ensures that we did not break anything and another that verifies the float handling that we added.